mod_reforged/msu_systems/tooltips.nut

```diff
@@ -22,5 +22,10 @@
 	},
 	Contract = {
 		FocusOnObjective = ::MSU.Class.BasicTooltip("Click to focus", "Click to focus on the objectives for this contract")
+	},
+	Tactical = {
+		Button = {
+			WaitTurnAllButton = ::MSU.Class.BasicTooltip("Wait Round", "Have all your men use \'Wait\' on their turn.")
```

Comment:
Have all your characters wait their turn

Reply:
I just found that vanilla actually uses \'Wait\' one time in their code so I will keep that part.

They use it in the perk description of Relentless:

> 	Relentless = "Don\'t slow down! At all times your Initiative is reduced only by [color=" + this.Const.UI.Color.NegativeValue + "]50%[/color] of your accumulated Fatigue, instead of all of it. In addition, using the \'Wait\' command will no longer give you a penalty to Initiative in the next round.",